docs/Design/SRS/SRS.tex

```diff
@@ -763,6 +765,129 @@ \subsubsection{Theoretical Models}\label{sec_theoretical}
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+\deftheory
+% #2 refname of theory
+{TM:Equilibrium}
+% #3 label
+{Equilibrium}
+% #4 equation
+{
+\( \sum{F_x} = 0, \sum{F_y} = 0, \sum{M}=0 \)
+}
+% #5 description
+{
+\begin{symbDescription}
+  \item{$F_x$ is the x-component of the force (\si{\newton})}
+  \item{$F_y$ is the y-component of the force (\si{\newton})}
+  \item{$M$ is the moment (\si{\newton\metre})}
+\end{symbDescription}
+}
+% #6 Notes
+{
+  For a body in static equilibrium, the net forces and moments acting on the body will cancel out. 
+  Assuming a 2D problem (\hyperref[A:Effective-Norm-Stress-Large]{A:Effective-Norm-Stress-Large}), the x-coordinate of the force $F_x$ and y-coordinate 
```

Comment:
should say the sum of x-coordinate of all forces and the same for y.

Reply:
Should we also add an assumption for *"Assuming a 2D problem"*

Reply:
Yes, we should have an assumption for 2D.  This is actually covered by the A:HalfPlane assumption that I already put in there.  :smile: